Plugins/org.mitk.cbica.captk.ui/src/QmitkExtWorkbenchWindowAdvisor.cpp

```diff
@@ -824,6 +824,12 @@ void QmitkExtWorkbenchWindowAdvisor::PostWindowCreate()
     berry::IPreferences::Pointer stylePrefs = prefService->GetSystemPreferences()->Node(berry::QtPreferences::QT_STYLES_NODE);
     bool showCategoryNames = stylePrefs->GetBool(berry::QtPreferences::QT_SHOW_TOOLBAR_CATEGORY_NAMES, true);
 
+    // CaPTk logo
+    prefService->GetSystemPreferences()->Node("org.mitk.editors.stdmultiwidget")->Put(
+      "DepartmentLogo", 
```

Comment:
@dboun what happened when you added this?

Reply:
Nothing changed. But maybe I put it in the wrong place?